azure-kusto-data/azure/kusto/data/kusto_client.py

```diff
@@ -197,6 +223,11 @@ class KustoClient(object):
     >>>    print(row["ColumnName"])    """
 
     def __init__(self, kusto_cluster, client_id=None, client_secret=None, username=None, password=None, version='v1', authority=None):
+        # review: why are you suddenly  documenting the individual parameters?
+        # And don't document the types unless you really mean it. And if you
+        # do then make them type hints and use mypy to help keep you honest.
+        # And does this follow any known docstring formatting style in the
+        # Python community (e.g. Sphinx, Google, etc.)?
```

Comment:
I used to do it, but it is not compatible with python 2.7

Reply:
@toshetah what's not compatible with Python 2.7? You can provide type hints for Python 2.7 if that's what you mean using a comment.